tests/unit_tests/data_steward/tools/add_hpo_test.py

```diff
@@ -122,3 +127,70 @@ def test_check_state_code_format(self):
 
         us_state = 'PIIState_XY'
         add_hpo.check_state_code_format(us_state)
+
+    def test_add_hpo_site_to_csv_files(self):
+        new_site = {
+            'org_id': 'fake_org',
+            'hpo_id': 'fake',
+            'hpo_name': 'fake_name',
+            'display_order': 3,
+            'bucket_name': 'fake_bucket'
+        }
+        hpo_site_csv_path = 'hpo_site_csv_files'
+
+        # This test will pass because by default, 'hpo_site_csv_files' folder is not present
+        # in Curation repo.
+        self.assertRaises(RuntimeError, add_hpo.add_hpo_site_to_csv_files,
+                          new_site['hpo_id'], new_site['hpo_name'],
+                          new_site['org_id'], new_site['bucket_name'],
+                          hpo_site_csv_path, new_site['display_order'])
```

Comment:
This is up to you: This test passing by default due to the lack of a folder may not be the most informative. For example, it would not have been able to find an error like the one mentioned above. I recommend adding a failure case of some kind as well.

Reply:
ok, sure.